UPDATING.md

```diff
@@ -22,7 +22,13 @@ under the License.
 This file documents any backwards-incompatible changes in Airflow and
 assists users migrating to a new version.
 
-## Airflow Master
+## Airflow 1.10.4
+
+### Python 2 support is going away
+
+Airflow 1.10 will be the last release series to support Pytho 2. Airflow 2.0.0 will only support Python 3.5 and up.
```

Comment:
```suggestion
Airflow 1.10 will be the last release series to support Python 2. Airflow 2.0.0 will only support Python 3.5 and up.
```

Reply:
D'oh :)

I force-pushed over Jarek's typo fixes.

Reply:
I'll wait for tests to pass, then fix-up the typos, then can live life on the end and merge without waiting for tests again.

Reply:
:D